config/locales/de/controllers/tasks.yml

```
@@ -9,15 +9,17 @@ de:
     export_external_confirm:
       error: 'Der Export der Aufgabe (%{title}) ist fehlgeschlagen. <br> Fehler: %{error}'
       success: Aufgabe (%{title}) erfolgreich exportiert.
+    import:
+      internal_error: Bei der Importierung der Aufgabe ist ein interner Fehler aufgetreten.
```

Just trying to improve the German wording a little; I know that you just moved it (due to the changed key):

```suggestion
      internal_error: Beim Import dieser Aufgabe ist auf CodeHarbor ein interner Fehler aufgetreten.
```